src/core/OmniFormElement.ts

```diff
@@ -0,0 +1,375 @@
+import { css, CSSResultGroup, html, LitElement, nothing, TemplateResult } from 'lit';
+export { ifDefined } from 'lit/directives/if-defined.js';
+import { property, query } from 'lit/decorators.js';
+import { ClassInfo, classMap } from 'lit/directives/class-map.js';
+import ComponentStyles from '../styles/ComponentStyles.js';
+
+/**
+ * Base class used by components to share common properties styles and functionality
```

Comment:
Is the description correct?

Reply:
Updated description to reference form controls.